### ISSUE TITLE
com.apple.desktoppicture grows

### ISSUE THREAD

**Issue by MoritzFago** [ifrins/Snappr#3]
because every desktop background is stored in /private/var/folders/9h/01x2ngvs2c3bv1ldq17t4h_80000gn/C/com.apple.desktoppicture

(maybe the path is slightly different on your mac) the folder grows to insane sizes.

**Comment by ifrins** [comment]
Hey! I've checked and I can reproduce this on my system (10.11). I believe OS X creates an "optimized" copy of the image for the screen resolution. 

I'm working on the 2.0 update, and I'll try to find if there's any kind of workaround to avoid such behavior, and otherwise, I'll make a radar to Apple.

**Comment by yesthisisjoe** [comment]
Hey @ifrins, were you ever able to find a fix for this issue?

**Comment by UniK** [comment]
Any news on that matter?
(7 GB and counting...)

**Comment by MoritzFago** [comment]
@UniK no, no news, but you can safely delete the folder, it'll regrow, so for the time you'll have to do that regularly.

At least thats my work-a-round

**Comment by ifrins** [comment]
Hey,

There are two things here:
1) There's a problem on Apple's end that they do not clean this directory. I've filled a radar with them [rdar://27393406](http://openradar.appspot.com/radar?id=6066399948570624)
2) There's a workarround that I'm trying to implement in Snappr so that the system doesn't generate new files in there, but, nevertheless Snappr won't delete the pictures that are stored there (it's a system folder so Snappr won't change the files in there). 

As @MoritzFago says, it seems it's safe to delete all the files (or at least the old files stored there).

Another question, has anyone been able to reproduce this on macOS Sierra?

**Comment by yesthisisjoe** [comment]
@ifrins I've been switching wallpapers at least 100 times on Sierra, but I only see 2 pictures there, both from within 24 hours.
